### ISSUE TITLE
[Bug] Folder permission is causing delay to play videos

### ISSUE THREAD

**Issue by MajinSaaY** [moneytoo/Player#429]
**Describe the bug**
Video takes some time to play after permission to a folder is granted. (maybe caused by searching for a subtitle?)

**To Reproduce**
1. Long press on the folder icon in player
2. Proceed with the dialog by pressing ok
3. give permission to a folder containing multiple video files, maybe 100 or more (folder with a few or no items will not cause this)
4. Open a video and see how long it takes to play (mine is 5-8 secs)

**Expected behavior**
No delay on opening a video.

**Device**
 - Model: Motorola One Fusion
 - OS: Android 11
 - Version: RPLS31.Q2-63-10-2-9

**Comment by moneytoo** [comment]
Yes, the subtitle search is indeed that slow. It can take a few seconds (if there are just dozens or hundereds of files) even on high end devices. This would really be good to improve. I could search for subtitles in the background and try to attach it but should I restore playback to that position or start it again from the beginning in case the user might miss a line?

**Comment by MajinSaaY** [comment]
what if it tries to find only one file with the same name as the video? would it be faster? I think vlc works that way.
